core/src/main/java/com/datastax/oss/driver/internal/core/type/codec/extras/time/ZonedTimestampCodec.java

```diff
@@ -13,19 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.datastax.oss.driver.internal.core.type.codec;
+package com.datastax.oss.driver.internal.core.type.codec.extras.time;
```

Comment:
I moved this codec to `com.datastax.oss.driver.internal.core.type.codec.extras.time` to align with other temporal codecs added in this PR.